.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 env:
     - MPLBACKEND='agg'
-    -
+    - GEOPANDAS='True'; MPLBACKEND='agg'
```

Comment:
make sure to add an option with none of these environment variables set. also feel free to drop python 3.4 from this